account/manager.go

```diff
@@ -480,14 +459,16 @@ func (m *Manager) resumeAccount(ctx context.Context, account *Account, // nolint
 		}
 
 		if createTx {
-			// We need a static sat/vByte value for the fee in
-			// SendOutputs, so we use the stored targetConf of the
-			// account to estimate it.
-			feeSatPerKw, err := m.cfg.Wallet.EstimateFee(
-				ctx, int32(fundingConfTarget),
-			)
-			if err != nil {
-				return err
+			if feeRate == 0 {
```

Comment:
See my comment above about removing this `if` by moving the fee estimation outside of this method.

Reply:
I will leave the `if feeRate == 0 {` as a sanity check and then return an error that "should never happen"